src/ALObject.ts

```diff
@@ -136,24 +195,25 @@ export class ALObject {
         let objectToken = new XliffIdToken();
         let parentNode = '';
         let parentLevel = 0;
-        objectToken.Id = this.objectId;
+        objectToken.id = this.objectId;
         objectToken.Name = this.objectName;
-        objectToken.Type = this.objectType;
+        objectToken.type = ObjectType[this.objectType];
         xliffIdWithNames.push(objectToken);
         let indentation = 0;
+        let currControl = new Control();
         for (let lineNo = 0; lineNo < lines.length; lineNo++) {
             const line = lines[lineNo].trim();
             let codeLine: NAVCodeLine = new NAVCodeLine();
-            codeLine.LineNo = lineNo;
-            codeLine.Code = line;
+            codeLine.lineNo = lineNo;
+            codeLine.code = line;
             if (lineNo === 0) {
-                codeLine.XliffIdWithNames = xliffIdWithNames.slice();
+                codeLine._xliffIdWithNames = xliffIdWithNames.slice();
             }
             const indentationIncrease = /{|{\s*\/{2}(.*)$|\bbegin\b\s*$|\bbegin\b\s*\/{2}(.*)$|\bcase\b\s.*\s\bof\b/i;
             let increaseResult = line.match(indentationIncrease);
             const indentationDecrease = /(}|}\s*\/{2}(.*)$|^\s*\bend\b)/i;
             let decreaseResult = line.match(indentationDecrease);
-            const xliffIdTokenPattern = /(\bdataitem\b)\((.*);.*\)|\b(column)\b\((.*);(.*)\)|\b(value)\b\(\d*;(.*)\)|\b(group)\b\((.*)\)|\b(field)\b\((.*);(.*);(.*)\)|\b(field)\b\((.*);(.*)\)|\b(action)\b\((.*)\)|\b(trigger)\b (.*)\(.*\)|\b(procedure)\b (.*)\(.*\)|\blocal (procedure)\b (.*)\(.*\)|\b(layout)\b|\b(actions)\b/i;
+            const xliffIdTokenPattern = /(\bdataitem\b)\((.*);.*\)|\b(column)\b\((.*);(.*)\)|\b(value)\b\(\d*;(.*)\)|\b(group)\b\((.*)\)|\b(field)\b\((.*);(.*);(.*)\)|\b(field)\b\((.*);(.*)\)|\b(part)\b\((.*);(.*)\)|\b(action)\b\((.*)\)|\b(trigger)\b (.*)\(.*\)|\b(procedure)\b (.*)\(.*\)|\blocal (procedure)\b (.*)\(.*\)|\b(layout)\b|\b(actions)\b/i;
```

Comment:
Alot of the regex could be pre-compiled and defined in an object for easier access and better performance. Not sure if I commented on this earlier in this PR so sorry if I'm repeating myself :)

Reply:
To-do in your future refactoring? :)